ts/client/src/client.ts

```diff
@@ -1258,15 +1336,18 @@ export class MangoClient {
       clientOrderId,
       limit,
     );
-
-    const ix2 = await this.serum3SettleFundsIx(
-      group,
-      mangoAccount,
-      externalMarketPk,
-    );
+    txes.push(...placeOrderIxes);
+    if (mangoAccount.getSerum3Account(serum3Market.marketIndex)?.openOrders) {
+      const ix2 = await this.serum3SettleFundsIx(
+        group,
+        mangoAccount,
+        externalMarketPk,
+      );
+      txes.push(ix2);
+    }
```

Comment:
Shouldn't we always do the settle funds ix after placing an order? What is this `if` check for?

Reply:
fixed